<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wPlaceObjects</class>
 <widget class="QMainWindow" name="wPlaceObjects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wPlaceObjects</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblObjectDensity">
        <property name="text">
         <string>Objects per mÂ²:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="hslObjectDensity">
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>75</number>
          </property>
          <property name="sliderPosition">
           <number>75</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbxObjectDensity">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>75</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblTerrainLayerID">
        <property name="text">
         <string>Terrain Layer ID:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSpinBox" name="sbxTerrainLayerID">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lblLayerTexture">
            <property name="text">
             <string>LayerTexture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblLayerTexturePicture">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTileSize">
        <property name="text">
         <string>Tile size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dsbxTileSize">
        <property name="suffix">
         <string notr="true"> m</string>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>600.000000000000000</double>
        </property>
        <property name="value">
         <double>300.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="gbxTiles">
        <property name="title">
         <string>Perform action on tiles...</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="lwgTiles">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTileInfo">
           <property name="text">
            <string>Supports only tiles which are listed in global.cfg file. If selection is empty, the action will be performed on whole map.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxObjects">
        <property name="title">
         <string>Objects</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QListWidget" name="lwgObjects">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="inputAddEditRemove" name="cuwObjects" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbxClearLayer">
        <property name="text">
         <string>Clear terrain layer painting after perform</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconQuit.svg</normaloff>:/rec/data/icons/iconQuit.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>inputAddEditRemove</class>
   <extends>QWidget</extends>
   <header>OTWidgets/inputaddeditremove.h</header>
   <container>1</container>
   <slots>
    <signal>addClicked()</signal>
    <signal>editClicked()</signal>
    <signal>removeClicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
